recipes/biofilm/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "biofilm" %}
+{% set version = "0.0.94" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/biofilm-{{ version }}.tar.gz
+  sha256: b285d80da26aebdf41c7b4273c1b31717a035ff731169092a4ca1ac98b2cff3d
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
```

```suggestion
  noarch: python
  skip: True   # [not linux]
```

Thanks @BastianZim for the info. @smautner, this change should make things happy.

---

Of course! This isn't actually needed, sorry, should've been more clear. The failure is fine and noarch can stay. We introduced this at some point to keep packages noarch, even if they aren't available on all platforms.

---

Since the package is built on Linux it's fine.

---

> Being encouraged to make recipes fail is a bizarre experience though.

That's because the CI here will always build on all OS so that we can see what is happening. In your feedstock, it will only build on Linux, and you won't (hopefully) see any errors.